app/containers/App/tests/index.test.js

```diff
@@ -61,4 +62,36 @@ describe('<App />', () => {
     expect(setLoggedInStatus).toHaveBeenCalledTimes(0);
```

Comment:
Rename 'setLoggedInStatus' if this is only called for setting the value to `false`.